service/Microsoft.Azure.Devices/HttpRegistryManager.cs

```diff
@@ -306,11 +306,13 @@ public override Task<Device> GetDeviceAsync(string deviceId, CancellationToken c
             return this.httpClientHelper.GetAsync<Device>(GetRequestUri(deviceId), errorMappingOverrides, null, false, cancellationToken);
         }
 
+        [Obsolete("Use CreateTwinQuery")]
```

Comment:
@yzhong94 @tameraw @jasmineymlo For naming, I prefer the "GetDevicesAsync" over CreateTwinQuery to obtain the device names. (The second doesn't seem to be at all related.)

Instead of obsoleting this, can we just create an overload that supports paging or change the internal implementation of this one?